### ISSUE TITLE
Bug: cursorline does not work

### ISSUE THREAD

**Issue by nxvu699134** [kyazdani42/nvim-tree.lua#306]
# Issue
After this https://github.com/kyazdani42/nvim-tree.lua/pull/287/commits/88fa67ad3175e18a083c63b0c3bccec6b7b8819f commit merged, auto set cursorline on NvimTree file does not work any more. Any problem with my config? TIA

### My config

```lua
g.nvim_tree_hide_dotfiles = 1
g.nvim_tree_git_hl = 1
g.nvim_tree_indent_markers = 1
g.nvim_tree_auto_close = 1
g.nvim_tree_follow = 1
g.nvim_tree_ignore = { '.git', 'node_modules', '.cache' }
g.nvim_tree_width_allow_resize  = 1
g.nvim_tree_add_trailing = 1

local tree_cb = require'nvim-tree.config'.nvim_tree_callback
vim.g.nvim_tree_bindings = {
  ["<CR>"]           = tree_cb("edit"),
  ["o"]              = tree_cb("edit"),
  ["v"]              = tree_cb("vsplit"),
  ["s"]              = tree_cb("split"),
  ["<BS>"]           = tree_cb("close_node"),
  ["<S-CR>"]         = tree_cb("close_node"),
  ["I"]              = tree_cb("toggle_ignored"),
  ["H"]              = tree_cb("toggle_dotfiles"),
  ["R"]              = tree_cb("refresh"),
  ["a"]              = tree_cb("create"),
  ["d"]              = tree_cb("remove"),
  ["r"]              = tree_cb("rename"),
  ["x"]              = tree_cb("cut"),
  ["c"]              = tree_cb("copy"),
  ["p"]              = tree_cb("paste"),
  ["q"]              = tree_cb("close"),
  ["<Nop>"]          = tree_cb("preview"),
  ["<Nop>"]          = tree_cb("prev_git_item"),
  ["<Nop>"]          = tree_cb("next_git_item"),
  ["<Nop>"]          = tree_cb("dir_up"),
  ["<Nop>"]          = tree_cb("full_rename"),
  ["<Nop>"]          = tree_cb("cd"),
  ["<Nop>"]          = tree_cb("tabnew"),
}
map('', '<leader>ls', ':NvimTreeToggle<CR>', {silent = true})
map('', '<leader>lf', ':NvimTreeFindFile<CR>', {silent = true})
cmd('autocmd FileType NvimTree setlocal cursorline')

```

**Comment by kyazdani42** [comment]
```
'cursorline' 'cul'	boolean	(default off)
			local to window
```
You gotta bind the command to `BufWinEnter`

**Comment by lockejan** [comment]
I ran into the same situation, because I set cursorline to only appear on my active window/buffer.
That made the cursorline only appear on the first time I toggled nvim-tree (see attached video).

Maybe it's worth adding a note to the readme.

Working example: `vim.cmd("autocmd BufWinEnter NvimTree setlocal cursorline")`


https://user-images.githubusercontent.com/25434434/127548805-2a329f9d-1962-4505-9114-1750a36c6b17.mov
